projects/origin-release/golang-1.10/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN yum install -y epel-release && \
 ADD *.rpm /tmp/origin-rpm/
 RUN yum install -y /tmp/origin-rpm/*.rpm && \
     yum clean all && \
-    curl https://storage.googleapis.com/golang/go$VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf - && \
+    curl https://storage.googleapis.com/golang/go$VERSION.linux-$(uname -m | sed 's/x86_64/amd64/g').tar.gz | tar -C /usr/local -xzf - && \
```

Comment:
Do subshells work in a `RUN`? I think I might prefer to see a variable here like `$VERSION` for `$ARCH`

Reply:
@stevekuznetsov Yeah, subshells work in RUN. (Ref: https://github.com/docker-library/golang/blob/caca81426c40bd592bcfcde80485af80faedeb31/1.12/buster/Dockerfile#L17) However, I have updated the PR to store arch value in `ARCH` variable and then use it for curl. Please let me know if that works. Thanks.